.github/workflows/release-develop.yml

```diff
@@ -1,21 +1,16 @@
 name: Budibase Prerelease
 concurrency: release-prerelease
 
-on: 
- push:
-    branches:
-      - develop
-    paths:
-      - '.aws/**'
-      - '.github/**'
-      - 'charts/**'
-      - 'packages/**'
-      - 'scripts/**'
-      - 'package.json'
-      - 'yarn.lock'
-      - 'package.json'
-      - 'yarn.lock'
- workflow_dispatch:
+on:
+  push:
+    tags:
+      - v*-alpha.*
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: "Release tag"
+        required: true
+        type: boolean
```

Comment:
Is this input used?

Reply:
If we want to trigger existing tagged releases. It might be useful for rollbacks, for example

Reply:
I mean more so that it doesn't look to be used in the pipeline? I could be missing something

Reply:
True, this will actually not work. The action already allows us selecting a tag:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/15987277/236465800-9eb48372-eb6a-4c43-87aa-539ebb164016.png">